lib/Inst/Inst.cpp

```diff
@@ -1174,16 +1174,68 @@ Inst *souper::getInstCopy(Inst *I, InstContext &IC,
   return Copy;
 }
 
+bool isTerminalInst(Inst *I) {
+  bool terminal;
```

Comment:
when writing this kind of function, don't use a state variable, just directly return the correct thing

Reply:
ok. will fix. I remember you have told me this thing before too but for some reason, I am naturally inclined to do it this way. I will change it.